upsidedownlist/bring-it-up
# Create categories filter

## Description
#### Summary:
Implement multi-select dropdown component to filter categories

#### How to test this PR:
1. `npm start` in the client folder
2. Dropdown will appear in the top left of the screen 
3. Click on the dropdown button to reveal options 
4. Press on the checkboxes to select options

## Screenshots:
<img width="204" alt="Screen Shot 2022-03-24 at 7 57 51 PM" src="https://user-images.githubusercontent.com/73853606/160047472-78b16d2a-c200-4953-9743-f362212e5f6a.png">
